migrations/tenant/0025-custom-metadata.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE storage.objects ADD COLUMN user_metadata jsonb NULL;
+ALTER TABLE storage.s3_multipart_uploads ADD COLUMN metadata jsonb NULL;
```

Comment:
this one needs to be metadata cos of the s3 protocol? cant we have this also as user_metadata?

Reply:
It doesn't have to be `metadata` i can rename it for consistency 👍

Reply:
Done